applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c

```diff
@@ -244,7 +244,7 @@ void le_audio_print_codec(const struct bt_audio_codec_cfg *codec, enum bt_audio_
 
 		ret = bt_audio_codec_cfg_get_chan_allocation(codec, &chan_allocation);
 		if (ret == -ENODATA) {
-			LOG_WRN("Channel allocation not available");
+			chan_allocation = 0;
```

Comment:
Add a comment that we are reverting to mono if no Channel_allocation is available